Packs/FortiSandbox/Integrations/FortiSandboxv2/test_data/url_rating_known.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": 14,
```

Comment:
looks like the issue in demistosdk is coming from this file and others that have the ids as an int. Can you switch these, hopefully it will make the validations pass

Reply:
No problem, swtiched.
This worked and I handled other errors, I only have the following left:
- image size: should I change it which will differ from v1?
- CommonDashboard version: Do we change this? if so what is it, minor, major?
- Release notes: I'm not sure I fully understand this one.

Reply:
hey yes, every pack you change needs to be released, otherwise even though you added a change to the pack it wont appear in the marketplace. this includes the commondashboards

Reply:
regarding imagesize we will take this in the internal pr